### PR TITLE
Fix tests + tab change filter reset

### PR DESCRIPTION
**Pull request by garyluu**
Fix for tests and also the tab change not triggering facet reset

Edit to add: follows #1127

### CODE REVIEW

**Comment by denis-yuen** [comment]
Assuming this is #2073

File: cypress/integration/manualTests/basic-enduser.ts [hunk 1]
```diff
@@ -100,14 +100,14 @@ describe('Test logged out home page', () => {
   it('home page search bar works', () => {
     cy.visit('/');
     cy.get('input[id=searchBar]').type('test{enter}');
-    cy.url().should('include', '/search?search=test');
+    cy.url().should('include', '/search?entryType=workflows&search=test');
```
Comment (denis-yuen) [inline cypress/integration/manualTests/basic-enduser.ts:103]: Thought, this is a test so not strictly responsible but got me thinking,
what happens with an old search that someone had bookmarked that doesn't specify an entry type?

Comment (garyluu) [reply cypress/integration/manualTests/basic-enduser.ts:103]: defaults to workflow with possibly weird facets selected

Comment (denis-yuen) [reply cypress/integration/manualTests/basic-enduser.ts:103]: I'm fine with defaults to workflows. Why weird facets selected though?

**Review by coverbeck — APPROVED** [review]

File: cypress/integration/manualTests/basic-enduser.ts (hunk 1 above)
Comment (garyluu) [reply cypress/integration/manualTests/basic-enduser.ts:103]: if you previously selected tool-specific facets and have a link for that, it'll now default to looking for a workflow with that facet selected...which won't yield any results anymore

Comment (denis-yuen) [reply cypress/integration/manualTests/basic-enduser.ts:103]: Oh ok, I think I can live with that

**Review by denis-yuen — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Pull request merged**